alibi/api/interfaces.py

```diff
@@ -67,12 +68,12 @@ class Explainer(abc.ABC):
     """
     Base class for explainer algorithms
     """
-    _version: ClassVar[str] = __version__
     meta = attr.ib(default=attr.Factory(default_meta), repr=alibi_pformat)  # type: dict
 
     def __attrs_post_init__(self):
-        # add a name to the metadata dictionary
+        # add a name and version to the metadata dictionary
         self.meta["name"] = self.__class__.__name__
+        self.meta["version"] = __version__
```

Comment:
The version needs to be added here instead of in the `default_meta` because each explainer has it's own `DEFAULT_META` defined in `api.defaults`.